src/cli/commands/wallet.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @module cli/commands
+ */
+
+import {CliCommand} from "./interface";
+import {CommanderStatic} from "commander";
+import logger from "../../logger";
+import fs from "fs";
+import {CliError} from "../error";
+import bls from '@chainsafe/bls-js';
+import { blsPrivateKeyToHex } from '../../util/bytes';
+
+interface IWalletCommandOptions {
+  outputFile: string;
+}
+
+export class CreateWalletCommand implements CliCommand {
+  public register(commander: CommanderStatic): void {
+    commander
+      .command("wallet")
+      .description("Generate wallet private key")
+      .option("-o, --outputFile [output_file]", "Path to output file destination", "keys/validator/bls.json")
+      .action(async (options) => {
+        // library is not awaiting this method so don't allow error propagation 
+        // (unhandled promise rejections)
+        try {
+          await this.action(options);
+        } catch (e) {
+          logger.error(e.message);
+        }
+      });
+  }
+
+  public async action(options: IWalletCommandOptions): Promise<void> {
+    if (fs.existsSync(options.outputFile)) {
+      throw new CliError(`${options.outputFile} already exists`);
+    }
+
+    const keyPair = bls.generateKeyPair();
```

Comment:
Might be useful to print private and public park to console

Reply:
I would probably avoid logging the private key, public key would make sense though

Reply:
so, change to log the public key and also include the public key also in saving the file?

Reply:
yes